akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -172,6 +172,7 @@ class ClusterSharding(system: ExtendedActorSystem) extends Extension {
     system.systemActorOf(Props[ClusterShardingGuardian].withDispatcher(dispatcher), guardianName)
   }
 
+  // no longer used, but kept for binary compatibility
   private[akka] def requireClusterRole(role: Option[String]): Unit =
```

Comment:
it could be removed because it's private[akka]

Reply:
`sbt validatePullRequest` has different opinion :)